draft-ietf-teep-protocol.md

```diff
@@ -1260,23 +1262,23 @@ token = 20
 {: numbered='no'}
 
 ~~~~
-82                        # array(2)
-   02                     # unsigned(2)
-   A4                     # map(4)
-      14                  # unsigned(20)
-      1A 77777777         # unsigned(2004318071, 0x77777777)
-      05                  # unsigned(5)
-      01                  # unsigned(1) within .size 8
-      06                  # unsigned(6)
-      00                  # unsigned(0) within .size 4
-      08                  # unsigned(8)
-      82                  # array(2)
-         81               # array(1)
-            4F            # bytes(16)
-               0102030405060708090A0B0C0D0D0F
-         81               # array(1)
-            4F            # bytes(16)
-               1102030405060708090A0B0C0D0D0F
+82                       # array(2)
+  02                     # unsigned(2)
+  A4                     # map(4)
+    14                   # unsigned(20)
+    1A 77777777          # unsigned(2004318071, 0x77777777)
+    05                   # unsigned(5)
+    01                   # unsigned(1) within .size 4
+    06                   # unsigned(6)
+    00                   # unsigned(0) within .size 4
+    08                   # unsigned(8)
+    82                   # array(2)
+      81                 # array(1)
+        4F               # bytes(15)
+          0102030405060708090A0B0C0D0D0F
```

Comment:
A bunch of places have ...0D0D0F instead of 0D0E0F.  Is that intentional?

Reply:
Thanks for catching them. All 0D0D0F are typo of 0D0E0F, I will revise the PR.

Reply:
I added a commit of fixing all 0D0D0F to 0D0E0F in this PR